07/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device=width, initial-scale=1.0">
+    <meta name="description" content="Venta de motos suzuki, Refacciones, mantenimientos y servicios correcctivos">
+    <meta name="author" content="Jesus David Ceballos Parra">
```

Comment:
I guess there is a missing relevant meta tag
---
Creo que hay una etiqueta meta relevante que hace falta